pypeclub/OpenPype
# Slack: notifications are sent with Openpype logo and bot name

## Brief description
Before this PR Slack notifications didn't contain any name nor icon. This adds that functionality.

## Additional info
For existing customers app must be reinstalled from `openpype\modules\slack\manifest.yml` and icon added in Slack app home page (see `website/docs/module_slack.md` for details)!

This PR is built on top of #2498 as it is touching similar parts of code and I was afraid of conflicts. Both functionality could be tested with this PR. 


## Testing notes:
1. Configure Slack notifications
2. Publish
3. Check that message contains name and icon
